src/components/swap/SwapButtons/SwapButton.tsx

```diff
@@ -24,23 +24,49 @@ const StyledSwapLoadingButton = styled(ButtonPrimary)`
   background-image: linear-gradient(90deg, #4c4c76 19.74%, #292942 120.26%);
   cursor: 'wait';
   padding: 16px;
+
+  ${({ theme }) => theme.mediaWidth.upToExtraSmall`
+    display: grid;
+    gap: 8px;
+    justify-items: center;
```

Comment:
Grid is super cool but it's unnecessary here.

`StyledSwapLoadingButton`inherits styles from `Base` and it has `flex` property

```suggestion
    flex-direction: column
```

Reply:
Makes sense Mikołaj, thanks!

Reply:
Fixed!